webapp/tests/mocha/unit/enketo/medic-xpath-extensions.spec.js

```diff
@@ -93,7 +99,7 @@ describe('medic-xpath-extensions', function () {
       ['2015-10-01T11:11:11.111', '2014-10-01T11:11:11.111', -12,],
       ['2014-10-01T00:00:00.000', '2015-10-01T00:00:00.000', 12,],
       ['August 19, 1975 00:00:00 GMT', 'August 18, 1976 23:15:30 GMT+07:00', 11],
-      ['Sun Sep 25 2005 1:00:00 GMT+0100', 'Sun Oct 25 2005 22:00:00 GMT+2300', 0],
+      ['Sun Sep 25 2005 1:00:00 GMT+0100', 'Sun Oct 25 2005 21:00:00 GMT+2300', 0],
```

Comment:
See this old thread for explanation of why this had to change:

https://github.com/medic/cht-core/pull/9017/files/e7ecd56957a509131c2df880b259bc8d5157a8a5#r1569328416